docs/en/reference/annotations-reference.rst

```diff
@@ -610,20 +610,17 @@ Examples:
 This annotation is used in the context of relations in
 :ref:`@ManyToOne <annref_manytoone>`, :ref:`@OneToOne <annref_onetoone>` fields
 and in the Context of :ref:`@JoinTable <annref_jointable>` nested inside
-a @ManyToMany. This annotation is not required. If it is not
-specified the attributes *name* and *referencedColumnName* are
-inferred from the table and primary key names.
+a @ManyToMany. If this annotation or both *name* and *referencedColumnName*
```

Comment:
Does the missing name and referenceColumnName mean a JoinColumn like this?

```
    /**
     * @ORM\OneToOne(targetEntity="App\Entity\Bar")
     * @ORM\JoinColumn(nullable=true)
     */
    public $bar;
```

Reply:
Yep